pkgs/top-level/python-packages.nix

```diff
@@ -563,6 +563,10 @@ in {
     inherit (pkgs) antlr4;
   };
 
+  antlr4_9-python3-runtime = callPackage ../development/python-modules/antlr4_9-python3-runtime {
```

Comment:
Probably not too important but I feel the general style is something like `antlr4_8-python3-runtime = ...; antlr4_9-python3-runtime = ...; antlr4-python3-runtime = antlr4_9-python3-runtime` (preferably the latest version).